CONTRIBUTING.md

```diff
@@ -62,10 +62,14 @@ intend only to run integration tests on a single package.
 
 #### GCP Setup
 
-To run the integrations tests, creation and configuration of two projects in
+To run the integrations tests, creation and configuration of three projects in
 the Google Developers Console is required: one specifically for Firestore
-integration tests, and another for all other integration tests. We'll refer to
-these projects as "general project" and "Firestore project".
+integration tests, one specifically for Bigtable integration tests, and another 
+for all other integration tests. We'll refer to these projects as 
+"general project", "Bigtable project" and "Firestore project".
```

Comment:
The order of these names should be the same as the descriptions in the previous sentence.

Reply:
Oh, yes. Corrected!